[docs] Delete superfluous bundles and update bundle docs

SplitInfinity opened the pull request:
**Description**
This commit removes the zfnet512 and vgg19 bundles because they are not
really necessary if there are two (resnet50 and lenet) working examples
of how to create standalone bundles. There are also some minor cmake
improvements.

**Testing**
ResNet50 bundles successfully build.

**Documentation**
This commit updates `docs/AOT.md` so that it explains the new `cmake` flow
for building standalone bundles.

**Issue**
This commit is part of #1979.

bertmaher commented on docs/AOT.md:
```diff
@@ -175,19 +171,18 @@ The makefile provides the following targets:
 ### Quantized network
-To build and run the example, you just need to execute:
-* `make run`. By default, quantized Resnet50 example is going to be executed.
+All of the aforementioned targets have quantized versions in CMakeLists.txt named `...QuantizedResNet50...`. For example, to run the quantized bundle, you just need to execute:
```
IMO the "named `...QuantizedResNet50...`" bit is a little confusing.  I think the example `ninja` command below is clear enough.

bertmaher commented on docs/AOT.md:
```diff
@@ -148,25 +148,21 @@ variables area.
 ### Floating point network
 To build and run the example, you just need to execute:
-* `QUANTIZE=NO make run`
+* `cmake ... -DGLOW_WITH_BUNDLES=ON ...`
```
The `...` is potentially confusing, I'd phrase it something like:
```
cmake -DGLOW_WITH_BUNDLES=ON  # ... other cmake flags ...
```

SplitInfinity commented:
- Updated references to `make` and Makefile in Overview section
- Addressed comments about unclear documentation

rdzhabarov commented on docs/AOT.md:
````diff
@@ -9,20 +9,18 @@ time.
 
 ```
-  $make
+  $cmake <other cmake flags> -DGLOW_WITH_BUNDLES=ON
````
can you add GLOW_WITH_CPU as well, since otherwise it does not work

rdzhabarov replied:
Or mention in the doc that compiling with CPU backend is required here.

rdzhabarov commented on docs/AOT.md:
```diff
@@ -148,25 +146,21 @@ variables area.
-You may need to adjust the environment variables at the top to match
-your setup, primarily LOADER and GLOW_SRC vars.
+* `cmake <other cmake flags> -DGLOW_WITH_BUNDLES=ON `
+* `ninja RunResNet50Bundle`
```
this only works if you use `Ninja` generator in the cmake command.

SplitInfinity commented:
Rebased and addressed @rdzhabarov 's comments:
- Added `-DGLOW_WITH_CPU=ON1` flag to all `cmake` commands in `AOT.md` because this is necessary for bundle compilation
- Added `-G ninja` flag to all `cmake` commands so that the commands that use `ninja` to run executables make sense
